src/doc/data_into_visit/SiloFormat.rst

```diff
@@ -1,6 +1,1621 @@
+The handle will be passed to other Silo_ function calls in order to add new objects to the file.
+Silo_ creates new files using the **DBCreate** function, which takes the name of the new file, access modes, a descriptive comment, and the underlying file type as arguments.
+In addition to being a library, Silo_ is a self-describing data model, which can be implemented on top of many different underlying file formats.
+Silo_ includes drivers that allow it to read data from several different file formats, the most important of which are: PDB (A legacy LLNL file format) format, and HDF5 format.
+Silo_ files stored in HDF5 format often provide performance advantages so the following code to open a Silo_ file will create HDF5-based Silo_ files.
+You tell Silo_ to create HDF5-based Silo_ files by passing the **DB_HDF5** argument to the **DBCreate** function.
+If your Silo_ library does not have built-in HDF5 support then you can pass **DB_PDB** instead to create PDB-based Silo_ files.
+
+
+Example for creating a new Silo file:
+
+.. tabs::
+
+  .. code-tab:: c
+
+    #include <silo.h>
+    #include <stdio.h>
+
+    int
+    main(int argc, char *argv[])
+    {
+        DBfile *dbfile = NULL;
+        /* Open the Silo file */
+        dbfile = DBCreate("basic.silo", DB_CLOBBER, DB_LOCAL,
+                          "Comment about the data", DB_HDF5);
+        if(dbfile == NULL)
+        {
+            fprintf(stderr, "Could not create Silo file!\n");
+            return -1;
+        }
+        /* Add other Silo calls here. */
+        /* Close the Silo file. */
+        DBClose(dbfile);
+        return 0;
+    }
+
+  .. code-tab:: fortranfixed
+
+        progam main
+        implicit none
+        include "silo.inc"
+        integer dbfile, ierr
+    c The 11 and 22 arguments represent the lengths of strings
+        ierr = dbcreate("fbasic.silo", 11, DB_CLOBBER, DB_LOCAL,
+        .               "Comment about the data", 22, DB_HDF5, dbfile)
+        if(dbfile.eq.-1) then
+            write (6,*) 'Could not create Silo file!\n'
+            goto 10000
+        endif
+    c Add other Silo calls here.
+    c Close the Silo file.
+        ierr = dbclose(dbfile)
+    10000 stop
+        end
+
+In addition to using the **DBCreate** function, the previous examples also use the **DBClose** function.
+The **DBClose** function ensures that all data is written to the file and then closes the Silo_ file.
+You must call the **DBClose** function when you want to close a Silo_ file or your file may not be complete.
+
+Dealing with time
+~~~~~~~~~~~~~~~~~
+
+Silo_ files are a flexible container for storing many types of data.
+Silo_'s ability to store data hierarchically in directories can allow you to store multiple time states of your simulation data within a single data file.
+However, since Silo is primarily an I/O library for storing files that contain a single time step's worth of data, VisIt_ only recognizes one time state per Silo_ file.
```

Comment:
This actually is not true but I also am not sure its worth delving into how to deal with this. A write can put all timesteps in a single silo file if they so choose. But, VisIt then needs to know that it needs to open the file **AND CD TO** a specific directory for each time step. We do support this, but its a rare case only useful for a long time series of small stuff so one doesn't generate a bazillion files.

Thoughts?

Reply:
I am fine with whatever you think is best. I was not the content provider here, just a translator from another format.

Reply:
```suggestion
However, Silo_ is most often used to store one time state per Silo_ file.
VisIt_'s Silo_ plugin is primarily designed and used to work with Silo_ files in this modality.
```

Reply:
Ok, I propose just a minor change here then.